components/frontapp/actions/send-new-message/send-new-message.mjs

```diff
@@ -0,0 +1,102 @@
+// legacy_hash_id: "a_poikPY"
```

Comment:
Would it be possible for you to remove the quotation marks around the action ID?

```
// legacy_hash_id: a_poikPY
```

It's not a big deal either way, but this will be marginally easier for me to parse

Reply:
Yes, that's an easy change.

Reply:
Removed quotation marks around the action ID